### PR TITLE
Finished all requirements in #31

### LINKED ISSUE #31: Community
• contains a background picture (changeable only by the person who created it)
• contains a join button for non-members
• contains a leave button for members
• contains a post button for members
• contains a delete button for creator/admins of the community
• contains a member count and you can click and view all the different member's names
• contains a description of the community
• contains a list of all community's posts ordered by latest timestamp first

### PR DESCRIPTION
**Pull request by qzho236**
# Description

• contains a background picture (changeable only by the person who created it)
• contains a join button for non-members
• contains a leave button for members
• contains a post button for members
• contains a delete button for creator/admins of the community
• contains a member count and you can click and view all the different member's names
• contains a description of the community
• contains a list of all community's posts ordered by latest timestamp first

## Related Issue

Solves Issue #31

## Type of change

- [ ] New feature (enhancement)

## How Has This Been Tested?

- [ ] Automated testing
- [ ] Manual testing

# Checklist:

- [ ] Does a similar (open or closed) pull request not already exist?
- [ ] Is the pull request _head_ repository a fork repository?
- [ ] Is the pull request _compare_ branch a development branch?
- [ ] Is the code documented, particularly in hard-to-understand areas?
- [ ] Does the code build without new warnings?
- [ ] Has testing been performed that proves changes are effective and work?
- [ ] Has a self- and/or peer-review of the code been performed?
- [ ] Have dependent changes been merged and published in downstream modules?
- [ ] Does all new and existing automated testing pass?
- [ ] Is the person responsible for the repository assigned to the pull request?
- [ ] Is the pull request linked to a project?
- [ ] Is the pull request linked to a milestone?

For more information, refer to the Contributing Guidelines and Code of Conduct links at the bottom of this page.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -4,35 +4,29 @@ import { Route, Routes } from "react-router-dom";
...
 import Homepage from "./pages/Homepage";
 import NewPost from "./pages/newPost/NewPost.jsx";
-import NewCommunity from "./pages/newCommunity/NewCommunity.jsx";
 import ForgotPasswordPage from "./pages/LoginRegister/ForgotPasswordPage";
 import LoginPage from "./pages/LoginRegister/LoginPage";
 import RegisterPage from "./pages/LoginRegister/RegisterPage";
 import NavBar from "./components/NavBar/NavBar";
-import CommunitiesPage from "./pages/Communities/CommunitiesPage";
 import Posts from "./pages/Posts/Posts";
+import Community from './pages/Community'
```
Comment (christopher-alba) [inline src/App.js]:
```suggestion
import CommunitiesPage from "./pages/Communities/CommunitiesPage";
import NewCommunity from "./pages/newCommunity/NewCommunity.jsx";
```

File: src/App.js [hunk 2]
```diff
@@ -4,35 +4,29 @@ import { Route, Routes } from "react-router-dom";
...
+    <Routes>
+      <Route exact path="/" element={<Homepage />} />
+      <Route exact path="/homepage" element={<Homepage />} />
+      <Route path="/login" element={<LoginPage />} />
+      <Route path="/register" element={<RegisterPage />} />
+      <Route path="/forgot-password" element={<ForgotPasswordPage />} />
+      <Route exact path="/ComponentsTesting" element={<ComponentsTesting />} />
+      <Route exact path="/NewPost" element={<NewPost />} />
+      <Route exact path="/Posts" element={<Posts />} />
+      <Route exact path="/Community" element={<Community />} />
```
Comment (christopher-alba) [inline src/App.js]:
```suggestion
      <Route exact path="/Community" element={<Community />} />
      <Route exact path="/NewCommunity" element={<NewCommunity />} />
      <Route exact path="/Communities" element={<CommunitiesPage />} />
```

File: src/index.css [hunk 3]
```diff
@@ -1,15 +1,22 @@
+@import '~antd/dist/antd.css';
```
Comment (christopher-alba) [inline src/index.css]:
```suggestion

```

**Review by christopher-alba — CHANGES_REQUESTED** [review]
Heya just a few changes so we can see the Netlify Preview, thanks!

File: src/App.js (hunk 1 above)
Comment (Xiaoqi510) [reply src/App.js]: i think his community is my communitiesPage, just need to replace it

File: src/App.js [hunk 4]
```diff
@@ -13,7 +12,9 @@ import NavBar from "./components/NavBar/NavBar";
 import Post from "./pages/Post";
 import CommunitiesPage from "./pages/Communities/CommunitiesPage";
 import Posts from "./pages/Posts/Posts";
-
+import Community from './pages/Community'
+import CommunitiesPage from "./pages/Communities/CommunitiesPage";
```
Comment (christopher-alba) [inline src/App.js]:
```suggestion

```

File: src/pages/Community/index.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
+import { Button } from 'antd';
```
Comment (christopher-alba) [inline src/pages/Community/index.js]:
```suggestion
import { Button } from '@mui/material';
```

File: src/pages/Community/index.js [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+function Community() {
+    return (
+        <div>
+            <div className='container detail-container'>
+                <div className='detail-card'>
+                    <div className='image' style={{ "backgroundImage": "url(https://img1.baidu.com/it/u=1407750889,3441968730&fm=253&fmt=auto&app=120&f=JPEG?w=1200&h=799)" }}>
+                        <div>
+                            <p>SOFTENG</p>
+                            <Button type='primary'>Join</Button>
```
Comment (christopher-alba) [inline src/pages/Community/index.js]:
```suggestion
                            <Button variant="filled">Join</Button>
```

**Review by christopher-alba — CHANGES_REQUESTED** [review]

File: src/pages/Community/index.js [hunk 7]
```diff
@@ -0,0 +1,59 @@
+import { Button } from '@mui/material';
+import './index.css';
+import NavBar from '../../components/NavBar/NavBar';
```
Comment (christopher-alba) [inline src/pages/Community/index.js]:
```suggestion

```

**Review by Xiaoqi510 — APPROVED** [review]
LGTM!

**Review by christopher-alba — APPROVED** [review]
For documentation purposes there are a few minor visual bugs that need to be fixed in this PR, but due to time constraints we were unable to fix them and will be passed down to team4 to fix and will be documented officially through an issue on Github. 

LGTM!

**Review by yuchendong13 — APPROVED** [review]
LGTM!

**Review by hpar461 — APPROVED** [review]
There are an obvious UI bugs and a very minor code refinements needed, but due to a time being, it was decided to be passed down to an upcoming team. 

Other than that, LGTM!

**Pull request merged**